### DISCUSSION TITLE
Creation of Cloud Foundry space in alternate location?

### DISCUSSION THREAD

**Discussion by dachtera** [SAP-archive/btp-setup-automator#271] [Q&A] [answered]
We are being told we can no longer create Cloud Foundry space in api.cf.eu10.hana.ondemand.com, and must select the API Endpoint: https://api.cf.eu10-004.hana.ondemand.com/ when setting up.  How we can select an alternate endpoint in the automator that is possible when manually creating the CF space?

**Comment by lechnerc77** [comment, marked as answer]
The specification of the CF landscape is currently not support. If you need this feature, please open a [feature request](https://github.com/SAP-samples/btp-setup-automator/issues/new?assignees=&labels=enhancement&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%3Ctitle%3E). PRs are welcome.

**Reply by lechnerc77** [reply]
Enhancement is provided with PR #275. See issue #274 for further details
